src/components/CommentList/CommentList.tsx

```diff
@@ -1,6 +1,24 @@
 import React from 'react';
 import './CommentList.scss';
+import { Comment } from '../../react-app-env';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
```

Comment:
I think you should replace interface with type. Because in our videos said, that interface we use for describing data, and type for our inner needs(for example Props).